.env.sample

```diff
@@ -20,7 +20,8 @@ HOMS_DB_NAME=homs
 HOMS_DB_USER=homs
 HOMS_DB_PASSWORD=homs
 HOMS_USER=user@example.com
-HOMS_PASSWORD=RENEWMEPLEASE
+HOMS_PASSWORD=changeme
+HOMS_TOKEN=RENEWMEPLEASE
 
 BPM_HOST=camunda
 BPM_PORT=8080
```

Comment:
This environment variable is set but is not used anythere. In docker-compose.yml fixed port number 8766 is used instead, and that's really strange because homs port is used properly.